### PR TITLE
chore: fix Import cognito resource on studio

### PR DESCRIPTION
**Pull request by ykethan**
#### Description of changes:
PR updates information on importing a cognito resource. modifies existing information on the resource.
images have been added to show users how to import cognito resource.

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [x] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [x] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br />
      _ref: MDX: `[link](https://link.com)`
            HTML: `<a href="https://link.com">link</a>`_
            
### When this PR is ready to merge, please check the box below
- [x] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/console/auth/import.mdx [hunk 1]
```diff
@@ -1,31 +1,41 @@
...
+It is best practice to create at least two app clients with the following conditions:
 
 - At least one “Web app client”: an app client without a client secret
 - At least one “Native app client“: an app client with a client secret
 
-The client secret is used by applications that have a server-side component that secure the client secret, which is why the native app client needs one.
+The client secret is used by applications that have a server-side component that secure the client secret, which is why the native app client needs one. If the Cognito user pool contains only one App Client, this will be used for both your web and mobile clients.
 
-To complete the import procedure, run the `amplify push` command.
+[Learn more about getting started with User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-cognito-user-pools.html) or [configuring a User Pool App Client](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html)
```
Comment (josefaidt) [inline src/pages/console/auth/import.mdx:33]:
```suggestion
[Learn more about getting started with Cognito User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-cognito-user-pools.html) or [configuring a User Pool App Client](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html)
```

File: src/pages/console/auth/import.mdx [hunk 2]
```diff
@@ -1,31 +1,41 @@
 export const meta = {
   title: `Import Amazon Cognito resources`,
-  description: `Configure Amplify Studio to use existing Amazon Cognito user pool and identity pool resources as an authentication and authorization mechanism for other Amplify categories ( For example, API, Storage, and more).`,
+  description: `Configure Amplify Studio to use existing Amazon Cognito user pool and identity pool resources as an authentication and authorization mechanism for other Amplify categories ( For example, API, Storage, and more).`
 };
 
-You can import existing Amazon Cognito resources into your Amplify project using Amplify Studio. To get started, choose **Authentication** from the **Set up** menu in your app's Studio. On the **Authentication** page, choose **Reuse existing Amazon Cognito resources**.
-
-This feature is particularly useful if you’re trying to do the following:
+Imported Amazon Cognito resource is particularly useful if you’re trying to do the following:
```
Comment (josefaidt) [inline src/pages/console/auth/import.mdx:6]:
```suggestion
Importing an Amazon Cognito resource is particularly useful if you’re trying to do the following:
```

File: src/pages/console/auth/import.mdx [hunk 3]
```diff
@@ -35,26 +45,34 @@ These roles are usually automatically configured when you create a new identity
 
 [Learn more about getting started with Identity Pools.](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-identity-pools.html)
 
-## Import and update your source code
+We can also use Amplify CLI to [import an Cognito identity pool resource](/cli/auth/import/).
```
Comment (johnpc) [inline src/pages/console/auth/import.mdx:48]: In order to import the identity pool, do you also need to import the user pool? Might be worth pointing that out if so.

Comment (johnpc) [reply src/pages/console/auth/import.mdx:48]: Looks like a typo "an Cognito identity pool resource" should be "a Cognito identity pool resource"

File: src/pages/console/auth/import.mdx [hunk 4]
```diff
@@ -1,31 +1,41 @@
...
-The client secret is used by applications that have a server-side component that secure the client secret, which is why the native app client needs one.
+The client secret is used by applications that have a server-side component that secure the client secret, which is why the native app client needs one. If the Cognito user pool contains only one App Client, this will be used for both your web and mobile clients.
 
-To complete the import procedure, run the `amplify push` command.
+[Learn more about getting started with Cognito User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-cognito-user-pools.html) or [configuring a User Pool App Client](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html)
 
-[Learn more about getting started with User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-cognito-user-pools.html)
-or 
-[configuring a User Pool App Client](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html)
+We can also use Amplify CLI to [import an Cognito user pool resource](/cli/auth/import/).
```
Comment (johnpc) [inline src/pages/console/auth/import.mdx:35]: typo - "an Cognito user pool"

**Review by johnpc — CHANGES_REQUESTED** [review]

**Review by johnpc — APPROVED** [review]

**Review by josefaidt — APPROVED** [review]

**Review by wrpeck — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**